cpp_tests/feature/one_hot_encoder_test.cc

```diff
@@ -20,45 +20,59 @@
 #include <sys/stat.h>
 
 #include "clink/cpp_tests/test_util.h"
+
 #include "clink/feature/one_hot_encoder.h"
 #include "nlohmann/json.hpp"
 #include "gtest/gtest.h"
 
-namespace tfrt {
+using namespace mlir;
```

Comment:
OK. I'll check all such usages.

Besides, a possibly better practice is to remove all `using namespace` statements, like stated in [why-is-using-namespace-std-considered-bad-practice](https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice). What do you think?

Reply:
I agree it makes sense to avoid using `using namespace` in general. Indeed, FRT has mostly followed this practice -- it only uses `using namespace mlir` in its codebase.